tools/docgen/docgen/__main__.py

```diff
@@ -7,6 +7,8 @@
 def instantiate_channel(channel_yml):
     name = channel_yml["name"]
     module_name, class_name = name.rsplit(".", 1)
+
+    module_name = f"docgen.{module_name}"
```

Comment:
this line might be a little problematic for people who want to make their own channels as plugins